src/navigation/sideNavConfig.js

```diff
@@ -22,6 +22,15 @@ const navigationItems = [
     title: 'API',
     subTitle: 'For developers',
     icon: 'Settings',
+    children: [
```

Comment:
It could be better if child nav groups are determined by `nav.path` in the frontmatter instead of in here.

Reply:
@AndyClifford I agree but I think this is still an improvement 🚀